CMakeLists.txt

```diff
@@ -1,28 +1,24 @@
-cmake_minimum_required(VERSION 3.2.2)
-set(CMAKE_INSTALL_PREFIX ${CMAKE_BINARY_DIR} CACHE PATH
-  "Locaction for installation"
-  FORCE)
-set(CMAKE_CONFIGURATION_TYPES Release Debug)
-set(CMAKE_CONFIGURATION_TYPES "${CMAKE_CONFIGURATION_TYPES}" CACHE STRING "The needed configuration" FORCE)
-project(TemplateExternRepo)
-include (version.cmake)
-enable_testing()
-set(externpro_DIR "C:/Program Files/externpro 18.01.1-vc140-64")
-list(APPEND XP_MODULE_PATH ${externpro_DIR}/share/cmake)
-include(${externpro_DIR}/share/cmake/xpfunmac.cmake)
-xpCommonFlags()
+cmake_minimum_required(VERSION 3.21)
+set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/cmake)
+include(preproject)
+project(TemplateProject VERSION 0.0)
+set(externpro_REV 22.02)
+find_package(externpro REQUIRED)
+xpSetFlags()
+xpFindPkg(PKGS boost)
+xpSourceDirIgnore(cmake)
+xpSourceDirIgnore(Test)
 
-set(CMAKE_CXX_STANDARD 14)
-set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+# set(CMAKE_CXX_STANDARD 14)
```

Comment:
Do you need these lines?

Reply:
`USE_FOLDERS` is in `preproject`
I have no idea if I need to set the C standard. I assume I don't, since it's working while this is commented out.